src/virtualenv/seed/embed/pip_invoke.py

```diff
@@ -17,6 +17,13 @@
 
 
 class PipInvoke(BaseEmbed):
+
+    packages = {
+        "pip": Version.latest,
```

Comment:
When using the legacy pip seeder, we always use the latest version. This will be latest version in bundled + search dir if running in no download mode, otherwise upstream latest.